ui/src/routes/Home/components/HomeView.js

```diff
@@ -108,61 +44,77 @@ class HomeView extends Component {
   }
 
   renderUserTypeButtons = (label, index) => {
-    let className = "user-select"
+    let className = 'user-select'
+    let radioDot = 'circle-o'
+    let ariaHid = false
     let checked = false
     if (this.props.survey && this.props.survey.userType === label) {
-      className = "user-select button-gradient-active"
+      className = 'user-select button-gradient-active'
+      radioDot = 'dot-circle-o'
+      ariaHid = true
       checked = true
     }
     return (
-      <div className={className}>
-        <label key={index}
-          htmlFor={label}>
-          <input key={index}
-            id={label}
-            onChange={(e) => this._onHandleUserTypeSelect(e)}
-            type="radio"
-            name="userType"
-            value={label}
-            checked={checked}
-            className="user-select__input"
-            ref={(input) => { this.inputField = input; }}
-            />
-          {label}
-        </label>
-      </div>
+      <label htmlFor={label} className={className}>
+        <Icon
+          name={radioDot}
+          className='user-select__radio-dot'
+          aria-hidden={ariaHid}
+          role='img'
+        />
+        <input key={index}
+          id={label}
+          onChange={(e) => this._onHandleUserTypeSelect(e)}
+          type='radio'
+          name='userType'
+          value={label}
+          checked={checked}
+          className='user-select__input visuallyhidden'
+          ref={(input) => { this.inputField = input }}
+        />
+        {label}
+      </label>
     )
   }
 
   renderUserCountButtons = (label, index) => {
-    let className = "count-select"
+    let className = 'count-num'
+    let radioDot = 'circle-o'
+    let ariaHid = false
     let checked = false
     if (this.props.survey && this.props.survey.userCount === label) {
-      className = "count-select button-gradient-active"
+      className = 'count-num button-gradient-active'
+      radioDot = 'dot-circle-o'
+      ariaHid = true
       checked = true
     }
     return (
-      <div className={className}>
-        <label key={index}
-          htmlFor={label}>
-          <input key={index}
-            onChange={(e) => this._onHandleUserCountSelect(e)}
-            type="radio"
-            name="userCount"
-            value={label}
-            id={label}
-            checked={checked} />
-          {label}
-        </label>
-      </div>
+      <label
+        htmlFor={label} className={className}>
+        <Icon
+          name={radioDot}
+          className='count-select__radio-dot'
+          aria-hidden={ariaHid}
+        />
+        <input key={index}
```

Comment:
You should also be able to remove `key` from the `input`, since it'll be unused...